### ISSUE TITLE
Server Console

### ISSUE THREAD

**Issue by dylan** [impact-crater/impact-crater-core#6]
e4248f7f0092cfe20b31edbd6babdf42ba415c34 Check out that commit, I'd like to know your thoughts about functionality. If you start the server with the -d flag you can see it, /help will list what's possible.

TODO
- kick
- ban
- direct message
- list clients
- connect avg ping
- connect total clients

Also right now server messages are going to client consoles.

**Comment by cha55son** [comment]
I'll be sure to check this out when I get home later. This is a really good idea, although I would probably shoot for something like **Extensible Server API**, then build the console on top of it. Either way this is cool.

**Comment by dylan** [comment]
Yeah, thinking about it more, some sort of plugin/adapter thing would be nice, and probably appropriate for this.
